### PR TITLE
Set powervs_cis_crn from install config and remove default

### PR DESCRIPTION
**Pull request by mjturek**

### CODE REVIEW

**Comment by clnperez** [comment]
looks okay so far. can you go ahead and add the survey question as well? even if this isn't the way ibmcloud does it in the end, generating this is something we can come back and add later.

**Comment by mjturek** [comment]
> looks okay so far. can you go ahead and add the survey question as well? even if this isn't the way ibmcloud does it in the end, generating this is something we can come back and add later.

Done!

File: pkg/types/powervs/platform.go [hunk 1]
```diff
@@ -11,6 +11,9 @@ type Platform struct {
 	// Required for multi-zone regions.
 	Zone string `json:"zone"`
 
+	// CRN of CIS Instance containing base domain
+	CISInstanceCRN string `json:"cis_crn"`
```
Comment (mkumatag) [inline pkg/types/powervs/platform.go:15]: I don't see IBMCloud folks got this extra entry into the Platform types, why don't we just follow the approach they are following.

Comment (mjturek) [reply pkg/types/powervs/platform.go:15]: Digging deeper, you're right, I'm going to refactor this a bit. Thanks!

**Comment by mjturek** [comment]
I'm going to abandon this as it's not the right approach. Submitting something closer to how IBM Cloud does things

**Pull request closed without merging**